app/views/conference_registrations/show.pdf.prawn

```diff
@@ -0,0 +1,148 @@
+prawn_document(filename: @pdf_filename, page_layout: :landscape, :page_size => [630.28,721.89]) do |pdf|
```

Comment:
the code should be move to `pysical_ticket#show`, right? and take into account, that admins and user should have the same view :wink: